modules/casper.js

```diff
@@ -254,11 +287,47 @@ Casper.prototype.back = function back() {
  * @param  String  url     The url to download
  * @param  String  method  The method to use, optional: default GET
  * @param  String  data    The data to send, optional
+ * @param  Boolean async   Asynchroneous request? (default: false)
  * @return string          Base64 encoded result
  */
-Casper.prototype.base64encode = function base64encode(url, method, data) {
+Casper.prototype.base64encode = function base64encode(url, method, data, async) {
     "use strict";
-    return this.callUtils("getBase64", url, method, data);
+    async = ((typeof(async) !== "undefined")? async : false);
+	if ((phantom.version.major > 1) && (!async)) {
```

Comment:
I strongly disagree, see my comments to this PR. Implicitly changing behavior due to different engine in use is evil imho. Better to explicitly tell the user that what was planned is obsolete / unsupported right away.